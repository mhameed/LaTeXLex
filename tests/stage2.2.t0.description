author: ehc
date: 09/04/03
description: check that we are removing special tab characters which we know can turn up outside the tabbing environment as spaces but leaving them alone when in the tabbing environment.


Note from 09/03/04:
From further reading about the tabbing environment I have decided not to write a test because I believe that we are dealing with the environment incorrectly. I have instead tried to determine what all the things do. This is a very complex environment. 