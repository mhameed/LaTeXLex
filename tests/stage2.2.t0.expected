Here is some text and then some maths:
$$
\psi : X \rightarrow Y
$$

And some text and now a tabbing env:
\begin{tabbing}
Here is some piece of text \= more text \= still more text \= last text \\
second row \>  more at one tab in \\
third row \> \> more at two tabs in
\end{tabbing}

\begin{verbatim}
Here is some text and then some maths:
$$
\psi\>: X \rightarrow Y
$$

And some text and now a tabbing env:
\begin{tabbing}
Here is some piece of text \= more text \= still more text \= last text \\
second row \>  more at one tab in \\
third row \> \> more at two tabs in
\end{tabbing}
\end{verbatim}
